packages/inter-protocol/src/contractSupport.js

```diff
@@ -78,7 +84,7 @@ export const checkDebtLimit = (debtLimit, totalDebt, toMint) => {
 };
 
 /**
- * @deprecated use makeMetricsPublishKit
+ * @deprecated not durable due to StoredSubscriber
```

Comment:
it's recommended to point to the replacement when known.

Reply:
yeah. in this case there's no direct replacement. basically DIY. to give instructions on what to would be quite long here, and duplicated for each